### PR TITLE
Add release notes for 2.19.3

### PR DESCRIPTION
**Pull request by abrookins**
Add release notes for 2.19.3.

### CODE REVIEW

**Review by zangell44 — APPROVED** [review]
LGTM!

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+### Enhancements
+- Provide URL in CLI output upon work pool creation — https://github.com/PrefectHQ/prefect/pull/13597
+- Allow flow parameter schema generation when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/13620
+
+### Fixes
+- Add handling for multi-word dbt CLI commands — https://github.com/PrefectHQ/prefect/pull/13616
+- Skip on cancellation hooks if runner cant load flow — https://github.com/PrefectHQ/prefect/pull/13669
+
+### Integrations
+- Add loading state to concurrency limits table — https://github.com/PrefectHQ/prefect-ui-library/pull/2483
```
Comment (serinamarie) [inline RELEASE-NOTES.md:14]:
```suggestion
- Add loading state to concurrency limits table in the Prefect UI — https://github.com/PrefectHQ/prefect-ui-library/pull/2483
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+- Remove old schema properties from deployments — https://github.com/PrefectHQ/prefect-ui-library/pull/2482
```
Comment (serinamarie) [inline RELEASE-NOTES.md:15]:
```suggestion
- Remove old schema properties from deployments in the Prefect UI — https://github.com/PrefectHQ/prefect-ui-library/pull/2482
```

Comment (serinamarie) [reply RELEASE-NOTES.md:15]: or maybe just the whole section could be called `UI` 🤷‍♀️

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
 # Prefect Release Notes
 
+## Release 2.19.3
+
+### Enhancements
+- Provide URL in CLI output upon work pool creation — https://github.com/PrefectHQ/prefect/pull/13597
+- Allow flow parameter schema generation when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/13620
+
+### Fixes
+- Add handling for multi-word dbt CLI commands — https://github.com/PrefectHQ/prefect/pull/13616
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]: I think this would actually go under integrations, perhaps?

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
 # Prefect Release Notes
 
+## Release 2.19.3
+
+### Enhancements
+- Provide URL in CLI output upon work pool creation — https://github.com/PrefectHQ/prefect/pull/13597
+- Allow flow parameter schema generation when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/13620
+
+### Fixes
+- Skip on cancellation hooks if runner can't load flow — https://github.com/PrefectHQ/prefect/pull/13669
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Skip on cancellation hooks if runner can't load flow — https://github.com/PrefectHQ/prefect/pull/13669
```
Maybe "Ensure graceful cancellation of flow runs corresponding to deleted deployments" ?

**Pull request merged**